src/pages/portfolio/AccountOverviewSection.tsx

```diff
@@ -96,7 +95,12 @@ export const AccountOverviewSection = () => {
     },
     (showVaults || (vaultBalance ?? 0) > 0.01) && {
       id: 'megavault',
-      label: stringGetter({ key: STRING_KEYS.MEGAVAULT }),
+      label: (
+        <Link onClick={() => handleViewVault()} withIcon tw="text-color-text-0">
```

Comment:
I don’t think we need the withIcon. That link out icon is typically used for navigating away from the app. i.e. help docs. Is the link not prominent enough with an underline? Could maybe also an arrow?